wrench/src/reftest.rs

```diff
@@ -47,6 +62,9 @@ pub fn parse_reftests<F>(filename: &str, mut runner: F)
         };
         let test = dir.join(items.next().unwrap());
         let reference = dir.join(items.next().unwrap());
+
+        assert!(test.is_file() && reference.is_file());
```

Comment:
Reftest parsing is very panicky right now. This at the least creates a better panic for a missing file than before. Eventually it would be nice to have error messages.